### ISSUE TITLE
Batch norm does not work

### ISSUE THREAD

**Issue by ArvidWeyrauch** [RAI-SCC/torch_blue#61]
As of January 2026 this is a known issue in PyTorch due to batch norm not being compatible with vmap. You can find official patching suggestions [here](https://docs.pytorch.org/functorch/stable/batch_norm.html).
